README.md

```diff
@@ -283,7 +284,9 @@ $ ls *.txt
 commands.txt  pi.txt
 ```
 
-This is a rather trivial example, but hopefully you see how powerful these tasks can be when trying to make your code easy for others to run.
+Further, you are not just limited to adding tasks through the command line, but can also edit the `pixi.toml` file's `[tasks]` block.
```

Comment:
We can now these the tasks we added previously are in `pixi.toml`. In addition to `pixi task add`, you can directly create tasks in `pixi.toml` if you so choose.

This is a rather trivial example, but hopefully you see how powerful tasks can be when trying to make your code easy for others to run. Not only can you run simple `python` scripts like in this example, but compile more complicated projects for languages that require such (C++ for example).